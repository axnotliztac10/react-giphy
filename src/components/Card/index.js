import React from 'react';
import { Card as RebassCard } from 'rebass';

export default function Card(props) {
  return (
    <RebassCard
      boxShadow="0 2px 16px rgba(0, 0, 0, 0.25)"
      borderRadius={8}
      bg="#fff"
      {...props}
    >{props.children}</RebassCard>
  );
}
