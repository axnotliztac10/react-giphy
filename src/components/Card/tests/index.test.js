import React from 'react';
import toJSON from 'enzyme-to-json';
import { shallow } from 'enzyme';

import Card from '../index';

describe('<Card />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Card><div /></Card>);

    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
