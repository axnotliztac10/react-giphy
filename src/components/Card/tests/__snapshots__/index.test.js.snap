// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> should render correctly 1`] = `
<Styled(styled.div)
  bg="#fff"
  borderRadius={8}
  boxShadow="0 2px 16px rgba(0, 0, 0, 0.25)"
>
  <div />
</Styled(styled.div)>
`;
